src/nvim/main.c

```diff
@@ -415,7 +415,13 @@ int main(int argc, char **argv)
 
   TIME_MSG("init default mappings");
 
-  init_default_autocmds();
+  err = ERROR_INIT;
+  o = NLUA_EXEC_STATIC("return vim._init_default_autocmds()",
```

Comment:
I'd suggest merge this with the preceding `NLUA_EXEC_STATIC` call to (1) avoid the repetition of boilerplate (2) entering/exiting the luajit VM is much more expensive than having a lua function just call another lua function.

Reply:
Thanks for the review. I was half-hoping that the diffs I was seeing were coming from a broken setup on my end, but it looks like they're real. I'll turn this PR into a draft, fix it, and then ping you again :)